test/FunctionalTests/Microsoft.OData.Client.Tests/metadata/ClientTypeUtilTests.cs

```diff
@@ -38,11 +38,12 @@ public void IFTypeProperty_HasNoKeyAttribute_TypeIsNotEntity()
             //Assert
             Assert.Equal(expectedResult, actualResult);
         }
+#endif
 
     }
     public class Person

```

Comment:
Could move this class inside the ClientTypeUtilTests class if you don't want to create it in its own file..